tests/integration/events/test_rabbitmq_event_publisher.py

```diff
@@ -37,3 +40,25 @@ def test_should_fail_publisher_when_connection_parameter_are_not_valid():
             service="service",
             topic="service-events",
         )
+
+@pytest.mark.integration
+@pytest.mark.skipif(
+    not rabbitmq_is_running_locally(), reason="RabbitMQ is not running locally"
+)
+def test_should_create_a_rabbitmq_event_publisher_and_publish_a_event_after_heartbeat(
+    make_user_created_event
+):
+    event = make_user_created_event()
+    rabbitmq_connector = RabbitMQConnector()
```

Comment:
It seems to me that the RabbitMQConnector is not instatiated on each test when running all the test suite.